bundler/lib/dependabot/bundler/file_fetcher/require_relative_finder.rb

```diff
@@ -33,7 +33,7 @@ def require_relative_paths
         sig { returns(Dependabot::DependencyFile) }
         attr_reader :file
 
-        sig { params(node: T.untyped).returns(T::Array[T.untyped]) }
+        sig { params(node: T.untyped).returns(T::Array[String]) }
```

Comment:
I'm curious and I don't know much about Sorbet... why does this go from `untyped` to `Array[String]`? I'm guessing the type didn't change, so why was it not previously set to `Array[String]`?

Reply:
Typically, we can set all types if all returns are converted to strings. However, as you mentioned, I noticed that we often use `T::Array[T.untyped]` in the system. I assume this is because Sobret was implemented later, and for quick solutions, people opted for this approach. Ideally, all types should be correctly identified and defined in the related methods. Personally, I use an IDE and generally try to follow its suggestions when possible.